x/dex/types/price.go

```diff
@@ -1,14 +1,20 @@
 package types
 
 import (
+	sdkerrors "cosmossdk.io/errors"
+
 	math_utils "github.com/neutron-org/neutron/v3/utils/math"
 	"github.com/neutron-org/neutron/v3/x/dex/utils"
 )
 
 // NOTE: 559_680 is the highest possible tick at which price can be calculated with a < 1% error
 // when using 26 digit decimal precision (via prec_dec).
 // The error rate for very negative ticks approaches zero, so there is no concern there
-const MaxTickExp uint64 = 559_680
+const (
+	MaxTickExp uint64 = 559_680
+	MinPrice   string = "0.000000000000000000000000495"
+	MaxPrice   string = "2020125331305056766452345.127500016657360222036663651"
```

Comment:
How was this number derived?

Reply:
this is just the price of 1.0001^559,680 and 1.0001^-559,680